app/server.go

```diff
@@ -245,6 +249,18 @@ func NewServer(options ...Option) (*Server, error) {
 		mlog.Error("Could not initiate logging", mlog.Err(err))
 	}
 
+	var imgErr error
+	s.imgDecoder, imgErr = imaging.NewDecoder(imaging.DecoderOpts{
+		ConcurrencyLevel: runtime.NumCPU(),
+	})
+	if imgErr != nil {
+		return nil, errors.Wrap(imgErr, "failed to create image decoder")
+	}
+	s.imgEncoder, imgErr = imaging.NewEncoder(imaging.EncoderOpts{})
```

Comment:
Just curious, are we not specifying a concurrency limit for encoding until we see a need for it?

Reply:
Good point. I think we can probably default to NumCPU as well. The way I wrote the PR I just forgot to add it.